grpc_client_node:
  ros__parameters:
    server_url: "127.0.0.1"
    server_port: 50051
    robot_id: "test_robot_1"
    robot_secret: "1234qwer"
    request_expired: 3.0
    server_ip: "127.0.0.1"
